### PR TITLE
Add missing `.env` variable

### PR DESCRIPTION
**Pull request by larsroettig**
Add `CHECKOUT_BRAINTREE_TOKEN` for env setup command

## Description

TODO: Describe your changes in detail here.

## Related Issue

Closes #ISSUE_NUMBER.

### Verification Steps

1. Go to the FOO page.
2. Verify the BAR shows up.
3. Make sure BAZ does a thing.

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.

### CODE REVIEW

File: pwa-devdocs/src/venia-pwa-concept/setup/index.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The subcommand generates a `packages/venia-concept/.env` file where you can set
 You can create the `.env` file and set the `MAGENTO_BACKEND_URL` value at the same time using a command similar to the following:
 
 ```sh
-MAGENTO_BACKEND_URL="https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/" yarn buildpack create-env-file packages/venia-concept
+MAGENTO_BACKEND_URL="https://master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud/" CHECKOUT_BRAINTREE_TOKEN="sandbox_8yrzsvtm_s2bg8fs563crhqzk" yarn buildpack create-env-file packages/venia-concept
```
Comment (sirugh) [inline pwa-devdocs/src/venia-pwa-concept/setup/index.md]: @larsroettig we intentionally removed this from the init code. Users should provide their own. At the least we suggest making this an obvious variable and calling that out.

Comment (larsroettig) [reply pwa-devdocs/src/venia-pwa-concept/setup/index.md]: Main problem in DOC is also a DEMO URL

Comment (sirugh) [reply pwa-devdocs/src/venia-pwa-concept/setup/index.md]: @jcalcaben what do you think? Should we add this into the documentation at least?

**Comment by jcalcaben** [comment]
Approved. The change should go live next week when we re-publish the site with the 5.0 docs. Thanks @larsroettig !

**Comment by jcalcaben** [comment]
hi @larsroettig we seem to be having issues with this PR. has the branch been deleted in your repository?

**Comment by larsroettig** [comment]
@jcalcaben i don't know what happens should I recreated a pr?

**Comment by jcalcaben** [comment]
@larsroettig yeah if you don't mind

**Comment by larsroettig** [comment]
> @larsroettig yeah if you don't mind

sure thank you and sorry

**Comment by larsroettig** [comment]
@jcalcaben https://github.com/magento/pwa-studio/pull/2153

**Pull request closed without merging**